packages/react-aria-components/test/Table.test.js

```diff
@@ -419,11 +557,11 @@ describe('Table', () => {
     expect(row).not.toHaveAttribute('data-pressed');
     expect(row).not.toHaveClass('pressed');
 
-    fireEvent.mouseDown(row);
+    fireEvent.pointerDown(row);
```

Comment:
fyi, i think these could be
```
await user.pointer({keys: '[MouseLeft>]', target: row});
await user.pointer({keys: '[/MouseLeft]', target: row});
```
https://testing-library.com/docs/user-event/pointer#pressing-a-button-or-touching-the-screen

Reply:
Unfortunately this issues a virtual event which our long press hook ignores, think I ran into the same thing in the test util library work I did

Reply:
o yeah... i ran into that as well, lets find a time to try to work through that, I want to make it easier for us to know which event we're firing